Gemfile

```diff
@@ -4,6 +4,7 @@ gem 'addressable'
 gem 'allowy', '>= 2.1.0'
 gem 'clockwork', require: false
 gem 'cloudfront-signer'
+gem 'concurrent-ruby'
```

Comment:
this was already a sub-dependency, but I wanted to make it explicit since this feature requires it